src/Microsoft.Data.Analysis/DataFrame.Arrow.cs

```diff
@@ -114,19 +127,21 @@ private static void AppendDataFrameColumnFromArrowArray(Field field, IArrowArray
                     break;
                 case ArrowTypeId.Timestamp:
                     {
-                        TimestampArray arrowTimeStampArray = (TimestampArray)arrowArray;
-                        var dataTimeDataFrameColumn = new DateTimeDataFrameColumn(fieldName, arrowTimeStampArray.Data.Length);
-                        for (int i = 0; i < arrowTimeStampArray.Data.Length; i++)
+                        var arrowTimeStampArray = (TimestampArray)arrowArray;
```

Comment:
We prefer the actual type and not `var`.

Reply:
Sure, will fix this. Does this include places where using the full type means repeating the type after a cast?
eg
```csharp
                        TimestampArray arrowTimeStampArray = (TimestampArray)arrowArray;
```

Reply:
Changed everywhere.